libs/sdk/e2e-schematics/src/generators/stories/index.ts

```diff
@@ -94,21 +91,21 @@ export default async function (tree: Tree, options: StoriesGeneratorSchema) {
     );
   }
 
-  // nx g @nrwl/angular:stories
+  // nx g @nx/angular:stories
   const optionsWithDefaults = {
     ...getGeneratorDefaults(
       tree,
-      '@nrwl/angular',
+      '@nx/angular',
       'stories',
       normalizedOptions.projectName
     ),
     name: normalizedOptions.projectName,
     cypressProject: normalizedOptions.cypressProject,
     generateCypressSpecs: normalizedOptions.generateCypressSpecs,
   };
-  angularStoriesGenerator(tree, optionsWithDefaults);
+  await angularStoriesGenerator(tree, optionsWithDefaults);
```

Comment:
The `angularStoriesGenerator` is async and without the `await` the files were not all being changed in the tree prior to looking for the tree changes in the later portion of this file.